### PR TITLE
Add triggers

### PR DESCRIPTION
**Pull request by gmadigan**
Added HLT_OldMu100 and HLT_TkMu100 triggers for 2017 and 2018.

### CODE REVIEW

**Comment by gmadigan** [comment]
Added NTupleInfo2017Full_stockNano.csv, includes correct event counts (using sum of weights) for 2017 stock NanoAODv6 samples for LQ2.

**Comment by gmadigan** [comment]
Fixed typos and corrected b-tag SF branch name (deepcsv)

**Comment by gmadigan** [comment]
Updated CMSSW version to CMSSW_10_2_18 when submitting condor jobs. Matches CMSSW version in LQ2 Twiki.

File: NTupleAnalyzer_nanoAOD.py [hunk 1]
```diff
@@ -2319,7 +2328,7 @@ def GetLLJJMasses3Jets(l1,l2,j1,j2,j3):
 	if (Branches['St_uujj'][0] < 275) and (Branches['St_uuj'][0] < 225): continue
 	#if (Branches['St_uujj'][0] < 275) and (Branches['St_uvjj'][0] < 275): continue
 	#if (Branches['M_uu'][0]    <  45) and (Branches['MT_uv'][0]   <  45): continue #this should be used if munujj channel is used
-        if (Branches['M_uu'][0]    <  45): continue #this should be used if munujj channel is not used
+		if (Branches['M_uu'][0]    <  45): continue #this should be used if munujj channel is not used
```
Comment (DavidMorse) [inline NTupleAnalyzer_nanoAOD.py]: This indentation looks wrong, can you check it please?

**Review by DavidMorse — CHANGES_REQUESTED** [review]

File: NTupleAnalyzer_nanoAOD.py (hunk 1 above)
Comment (gmadigan) [reply NTupleAnalyzer_nanoAOD.py]: Yeah, that should be fixed--actually, the whole skim needs to be corrected (I did this manually when running). Check out my new commit.

**Comment by gmadigan** [comment]
There was some weird indentation I fixed, plus I the fixed the skimming. Also, there was code that had been commented out in the HH version involving the creation of junk files. I think it runs without--is it necessary to keep?

**Comment by DavidMorse** [comment]
No, you should not remove the junkfiles - they are a pre-skim to reduce the running time. Its commented out in HH because the offline pt cuts are too low to make a useful skim. Please put it back in

**Comment by gmadigan** [comment]
The pre-skim has been added back.

File: NTupleAnalyzer_nanoAOD.py [hunk 2]
```diff
@@ -2175,7 +2175,7 @@ def GetLLJJMasses3Jets(l1,l2,j1,j2,j3):
 	#	Branches['weight_pu_up'][0]*=t.amcNLOWeight
 	#	#Branches['weight_central_2012D'][0]*=t.amcNLOWeight
 	#	Branches['weight_nopu'][0]*=t.amcNLOWeight
-	Branches['weight_amcNLO'][0]=0#t.amcNLOWeight
+	#Branches['weight_amcNLO'][0]=0#t.amcNLOWeight
```
Comment (DavidMorse) [inline NTupleAnalyzer_nanoAOD.py]: Now this variable is defined in the weights at the beginning but never set to anything. It should either be removed completely around line 170, or left to be set to 0 here.

File: NTupleAnalyzer_nanoAOD.py [hunk 3]
```diff
@@ -2312,14 +2321,15 @@ def GetLLJJMasses3Jets(l1,l2,j1,j2,j3):
...
-	if nonisoswitch != True:
-			if (Branches['Pt_muon2'][0] < 45) and (Branches['Pt_miss'][0] < 45): continue
+	#if nonisoswitch != True:
+	#		if (Branches['Pt_muon2'][0] < 45) and (Branches['Pt_miss'][0] < 45): continue
 	if (Branches['Pt_jet1'][0] <  45): continue
-	#if (Branches['Pt_jet2'][0] <  45): continue #fixme turned off for qcd check.....turn back on!
-	if (Branches['St_uujj'][0] < 275) and (Branches['St_uuj'][0] < 225): continue
+	if (Branches['Pt_jet2'][0] <  45): continue #fixme turned off for qcd check.....turn back on!
+	if (Branches['St_uujj'][0] < 275): continue
+	#if (Branches['St_uujj'][0] < 275) and (Branches['St_uuj'][0] < 225): continue
```
Comment (DavidMorse) [inline NTupleAnalyzer_nanoAOD.py]: Turn this back on, and comment out line 2328. This is to include the single LQ channel

**Review by DavidMorse — CHANGES_REQUESTED** [review]

File: NTupleAnalyzer_nanoAOD.py (hunk 3 above)
Comment (gmadigan) [reply NTupleAnalyzer_nanoAOD.py]: Right. I think that variable hadn't been added at some point, so I started commenting it out. I see it's there now, though.

**Pull request merged**